packages/core-commands/src/site-editor-navigation-commands.js

```diff
@@ -170,7 +171,7 @@ const getNavigationCommandLoaderPerTemplate = ( templateType ) =>
 				return {
 					isBlockBasedTheme:
 						select( coreStore ).getCurrentTheme()?.is_block_theme,
-					canCreateTemplate: select( coreStore ).canUser( 'create', {
+					canCreateTemplate: select( coreStore ).canUser( 'read', {
```

Comment:
Why did we change the perm? Seems natural to check for create in `canCreate...`

Reply:
Why? The commands are meant to navigate to templates, not _create_ new templates?

Reply:
But yes this variable should be renamed 😃

Reply:
I agree that using the `create` capability check here is odd but necessary. Users who can read templates might not have access to the Site Editor.

See https://github.com/WordPress/gutenberg/pull/60317#issuecomment-2156416761.

P.S. We should extract similar minor and unrelated fixes into separate PRs. There's already a lot going on here and that would make reviewing bit easier.

Reply:
Then this check should be called differently and we should use something else to check if the user has access to the site editor. This is not a good proxy for checking site editor access, it is more restrictive now. Also we should then use it as a condition to prevent all site editor commands from loading?

Reply:
>  Also we should then use it as a condition to prevent all site editor commands from loading?

It's hard to generalize it. The Site Editor needs more granular capability checks, but those should be mapped out separately - maybe even incorporated into the router.

As I mentioned, I would remove and extract similar fixes into separate PRs.

Reply:
Yes, here the change is necessary because you cannot `create` static templates through the endpoint, you can only create custom templates.

Reply:
So I guess we need another additional check, not sure what